.gitignore

```diff
@@ -231,3 +231,8 @@ tools/LinuxOneShot/Database
 tools/LinuxOneShot/TGS_Config
 tools/LinuxOneShot/TGS_Instances
 tools/LinuxOneShot/TGS_Logs
+/.vs/SS13KF/v16/Browse.VC.db
+/.vs/SS13KF/v16/Browse.VC.db-shm
+/.vs/SS13KF/v16/Browse.VC.db-wal
+/.vs/SS13KF/v16/Browse.VC.opendb
+/.vs/slnx.sqlite
```

Comment:
What are these for?

Reply:
Those are files that Visual studio automatically added to the project upon me loading it, not sure how to remove. I tried a couple things to make it so I wouldn't have to change the gitignore file to get them to not be picked up, but none of them worked. I thought of one more thing I can try, and if that works i will redo this